docs/core/tutorials/with-visual-studio.md

```diff
@@ -103,6 +103,7 @@ Enhance the application to prompt the user for their name and display it along w
    This code displays a prompt in the console window and waits until the user enters a string followed by the <kbd>Enter</kbd> key. It stores this string in a variable named `name`. It also retrieves the value of the <xref:System.DateTime.Now?displayProperty=nameWithType> property, which contains the current local time, and assigns it to a variable named `date` (`currentDate` in Visual Basic). And it displays these values in the console window. Finally, it displays a prompt in the console window and calls the <xref:System.Console.ReadKey(System.Boolean)?displayProperty=nameWithType> method to wait for user input.
 
    The `\n` (or `vbCrLf` in the Visual Basic code) represents a newline character.
+   The `Environment.NewLine` is another way to represent a newline character, but is platform independent and works in both C# and Visual Basic code. This is further described in the [Environment.NewLine](https://docs.microsoft.com/dotnet/api/system.environment.newline) document.
```

Comment:
```suggestion
   <xref:System.Environment.NewLine> represents a newline character.
```

Reply:
Is there another place you think I should add details in the tutorial about what `Environment.NewLine` is and what it does in a short summary, as I'm thinking if you are coming from something like C/C++ you may be used to seeing `\n` and not `Environment.NewLine` and might not know what the advantage to this way of representing a newline character vs `\n`.
As well as this, having a comment to say something like "We use `Environment.NewLine` instead of `\n` or `vbCrLf` for __this___ reason" could be helpful to guide usage towards using `Environment.NewLine` even if they see `\n` or `vbCrLf` elsewhere.

Reply:
That makes sense, how about something closer to your original text, like this:

> <xref:System.Environment.NewLine> is a platform-independent and language-independent way to represent a line break. Alternatives are `\n` in C# and `vbCrLf` in Visual Basic.

For more details, the reader can click on the API ref link.